bigtable/google/cloud/bigtable/mutation.py

```diff
@@ -0,0 +1,125 @@
+from google.cloud._helpers import _microseconds_from_datetime
+from google.cloud._helpers import _to_bytes
+from google.cloud.bigtable_v2.proto import (
+    data_pb2 as data_v2_pb2)
+from google.cloud.bigtable_v2.proto import (
+    bigtable_pb2 as table_v2_pb2)
+
+
+class RowMutations(object):
+    """Create Entry using list of mutations
```

Comment:
I think that the comments need to be fleshed out a bit more.  Let's discuss this offline.